docs/tutorials/getting_started/laziness.md

```diff
@@ -20,11 +20,6 @@ This tutorial will show both you how to create `DatasetReader`s
 that allow this lazy behavior, and how to handle this laziness
 when training a model.
 
-If you're not interested in the details,
```

Comment:
Not sure why you're changing this in this PR.

Reply:
Because I've moved 90% of the tutorials to live in the `docs`, and this one was unnecessary. It made sense to delete it.

Reply:
:+1: